lib/server-lib.js

```diff
@@ -43,6 +43,12 @@ function isConnected(remote) {
     // transactions
     return false;
   }
+  
+  if (remote._stand_alone){
```

Comment:
This may be ok, but it should be moved down.

```js
if (!server) return false
if (remote._stand_alone) return true
return ledgerCloseCheck()
```